lib/ansible/plugins/inventory/netbox.py

```diff
@@ -65,6 +65,23 @@
   - device_roles
 query_filters:
   - role: network-edge-router
+
+# Query filters are passed directly as argument to the fetching queries.
+# You can repeat tags in the query string.
+# Also, some more documentation and examples for query_filters would be good.
+
+query_filters:
+  - role: server
+  - tag: web
+  - tag: production
+
+# As the documentation of NetBox specify it https://netbox.readthedocs.io/en/latest/api/overview/
```

Comment:
remove this line, or limit it to "See the NetBox documentation at https://netbox.readthedocs.io/en/latest/api/overview/."